packages/design/src/styles/dark.mode.suffixStyles.css

```diff
@@ -1,5 +1,13 @@
-/* autoprefixer: off */
```

Comment:
Was this autoprefixer comment doing anything previously that we don't need it anymore?

Reply:
Hmmm I think I should add it back since we are keeping the current code for the safari use case 👍  Thanks 😅